apps/editors/tests/test_views.py

```diff
@@ -2970,17 +2970,19 @@ def generate_files(self, subset=[]):
         return results
 
     def test_results(self):
-        #import ipdb; ipdb.set_trace()
+        self.expected_addons = [addon for addon
+                                in self.generate_files().values()
+                                if addon.latest_version.nomination <=
+                                datetime.now() -
```

Comment:
this would be easier to read if it was indented on this line. The whole thing would probably be more legible if you created a new list in a loop rather than all on one line in a comprehension.